README.md

```diff
@@ -88,6 +88,10 @@ Where each location is on each platform:
     * bin   = bin
     * share = .
     * user  = .
+* Windows installed:
+    * $bin   = C:\Program Files\Minetest\bin
+    * $share = C:\Program Files\Minetest
+    * $user  = C:\Users\<user>\AppData\Roaming\Minetest
```

Comment:
Shouldn't we rather use something like `%APPDATA%\Minetest` respectively `%PROGRAMFILES%\Minetest` for these paths?

Reply:
As you can see in https://github.com/minetest/minetest/pull/6153/files#diff-59f99d02fe5b328bb9a57b023497b5edR367 It does automatically resolve the %APPDATA% path. 
And the Installer does also resolve the %PROGRAMFILES% variable and uses this as default.

This is only documentation, but i think a 10 year old will better understand the full written path instead of %APPDATA%\Minetest. A experienced user that changed the Environment variable will also understand that his data is then in his custom path. But I can also change this part of documentation, its on your opinion.

Reply:
`%PROGRAMFILES%\Minetest` and `%APPDATA%\Minetest` can be entered in the Windows Explorer path field (or in the `Windows+R` Run dialog) and Windows will replace the variable and access the correct path.

Reply:
Im sorry, I changed this some days ago, but It did not sucessfully push (probably due to network trouble).

I only changed $user to %Appdata%\Minetest because %ProgramFiles%\Minetest will not always resolve to the correct path:
 a) User installed it somewhere else
 b) User installed the x86 version on x64 Windows (WOW64)